db/seeds.rb

```diff
@@ -16,3 +16,7 @@
 5.times do |k|
   Reward.create(title: "Reward #{k+1}", description: "A reward.", price: k+1)
 end
+
+4.times do |l|
+  Order.create(reward_id: "#{l+1}", employee_id: "#{l+1}", purchase_price: "#{l+1}")
```

Comment:
Maybe instead of creating a fixed id for rewards and employees, we could sample them from existing objects.
```suggestion
  Order.create(reward_id: Reward.all.sample.id, employee_id: Employee.all.sample.id, purchase_price: "#{l+1}")
```

Reply:
done :)